### PR TITLE
[MB-1818] Release 1.1.0

### PR DESCRIPTION
**Pull request by z0br0wn**
Updates to Android SDK 7.2.0 and makes the Extender interface public.

### CODE REVIEW

**Comment by rlepinski** [comment]
👍

File: build.gradle [hunk 1]
```diff
@@ -15,9 +15,6 @@ buildscript {
 allprojects {
     repositories {
         jcenter()
-        maven {
-            url "http://dl.bintray.com/urbanairship/android"
```
Comment (rlepinski) [inline build.gradle]: this is still needed for now

**Comment by linkmh** [comment]
:shipit:

**Comment by hcrowell** [comment]
🍊

**Pull request merged**